src/devtools/client/webconsole/components/App.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+
+// We directly require Components that we know are going to be used right away
+import ConsoleOutput from "devtools/client/webconsole/components/Output/ConsoleOutput";
+import FilterBar from "devtools/client/webconsole/components/FilterBar/FilterBar";
+import { ConsoleNag } from "ui/components/shared/Nags/Nags";
+import { FilterDrawer } from "./FilterDrawer";
+import Warning from "ui/components/shared/Warning";
+import { FC, MouseEventHandler, useEffect } from "react";
+import { useSelector } from "react-redux";
+import JSTerm from "devtools/client/webconsole/components/Input/JSTerm";
+
+/**
+ * Console root Application component.
+ */
+const App: FC = () => {
+  // @ts-ignore
```

Comment:
:question: What TS errors are you getting here?  Is it just that TS doesn't know what type `state` is?

Reply:
Yeah, this state is still in JS files. Didn't want to bloat this PR with extra changes since this is already coming from another PR as a partial update that can be quickly merged.